storage/tool/pom.xml

```diff
@@ -44,10 +44,19 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Any reason for adding commons-io ?

Reply:
I am using it ShellMigrationResolver.java where we build checksum of the migration from the content of the file read as a byte array. To read the content as byte array I am using IOUtils from commons-io.